opensrp-reveal/src/main/java/org/smartregister/reveal/util/RevealJsonFormUtils.java

```diff
@@ -259,4 +259,12 @@ private void populateField(JSONObject formJson, String key, String value) throws
             field.put(VALUE, value);
         }
     }
+
+    public void populateField(JSONObject formJson, String key, String value, String fieldToPopulate) throws JSONException {
```

Comment:
This looks like a duplicate of the above method, could you change the method above to recall this method with `VALUE` as last param